modules/k6/k6.go

```diff
@@ -17,17 +21,78 @@ type K6Container struct {
 	testcontainers.Container
 }
 
+type RemoteTestFileDescription struct{
+
+	Uri string
+	DownloadDir string
+	User string
+	Password string
+}
+
+func ( d *RemoteTestFileDescription) getDownloadPath() string{
+	baseName := path.Base(d.Uri)
+	return path.Join(d.DownloadDir,baseName)
+
+}
+
+func downloadFileFromDescription(d RemoteTestFileDescription) error{
+
+	client := http.Client{Timeout: time.Second*60}
+	// Set up HTTPS request with basic authorization.
```

Comment:
I think this comment should go here: https://github.com/testcontainers/testcontainers-go/pull/2350/files#diff-19a4c472d6cc0154e74ab208f0a048d900eb0e6006b4365b853d56218d73cd5fR48, next to where it applies